runtime-display/libxnvctrl/autobuild/build

```diff
@@ -0,0 +1,9 @@
+abinfo "Building libXNVCtrl.so ..."
+cd "$SRCDIR"/src/libXNVCtrl/
+export OUTPUTDIR="out"
```

Comment:
Why was this necessary?

Reply:
Instead of building the entire `nvidia-settings`, we can build `libxnvctrl` independently.